tests/align/talign.nim

```diff
@@ -39,15 +43,13 @@ proc foobar() =
   doAssert (cast[uint](addr(mylocal2)) and 127) == 0
   doAssert (cast[uint](addr(mylocal3)) and 127) == 0
 
-  echo "align ok"
```

Comment:
This echo is completely useless right? it is already preceded by doAsserts, and is not dependent on any computed value.

Reply:
Besides it's usage as a progress indicator during debugging, I don't think it serves any purposes, so it's removal is fine.